### PR TITLE
add prefix to hide column avoiding conflict

### PR DESCRIPTION
**Pull request by jianwan0214**
## What type of PR is this?

- [ ] API-change
- [x] BUG
- [ ] Improvement
- [ ] Documentation
- [ ] Feature
- [ ] Test and CI
- [ ] Code Refactoring

## Which issue(s) this PR fixes:

issue #5448

## What this PR does / why we need it:
add prefix to hide column avoiding conflict

### CODE REVIEW

**Review by ouyuanning — APPROVED** [review]

File: pkg/vm/engine/tae/catalog/model.go [hunk 1]
```diff
@@ -34,7 +34,7 @@ var (
 )
 
 const (
-	PhyAddrColumnName    = "PADDR"
+	PhyAddrColumnName    = "__mo_PADDR"
```
Comment (nnsgmsone) [inline pkg/vm/engine/tae/catalog/model.go:37]: Why is the definition inside the tae. Shouldn't this be as common as catalog?

Comment (jianwan0214) [reply pkg/vm/engine/tae/catalog/model.go:37]: I find this const definition here, then use this const value.

**Review by daviszhen — APPROVED** [review]

**Review by aunjgr — APPROVED** [review]

File: pkg/sql/plan/build_show.go [hunk 2]
```diff
@@ -18,14 +18,16 @@ import (
...
+
 	"github.com/matrixorigin/matrixone/pkg/catalog"
 	"github.com/matrixorigin/matrixone/pkg/common/moerr"
 	"github.com/matrixorigin/matrixone/pkg/container/types"
 	"github.com/matrixorigin/matrixone/pkg/pb/plan"
 	"github.com/matrixorigin/matrixone/pkg/sql/parsers"
 	"github.com/matrixorigin/matrixone/pkg/sql/parsers/dialect"
 	"github.com/matrixorigin/matrixone/pkg/sql/parsers/tree"
-	"strings"
+	catalog2 "github.com/matrixorigin/matrixone/pkg/vm/engine/tae/catalog"
```
Comment (fengttt) [inline pkg/sql/plan/build_show.go:30]: Sigh ...   catalog2.   No comment.

File: pkg/vm/engine/tae/catalog/model.go (hunk 1 above)
Comment (fengttt) [reply pkg/vm/engine/tae/catalog/model.go:37]: Move it out to common catalog.

**Review by fengttt — COMMENTED** [review]
Is PADDR the only system column?    Others like composite primary key?

External table, may want to add columns like __mo_filename, __mo_time, etc.   So this list may grow.   Better keep it in one good place.

**Comment by jianwan0214** [comment]
> Is PADDR the only system column? Others like composite primary key?
> 
> External table, may want to add columns like __mo_filename, __mo_time, etc. So this list may grow. Better keep it in one good place.

ok

File: pkg/vm/engine/tae/catalog/model.go (hunk 1 above)
Comment (jianwan0214) [reply pkg/vm/engine/tae/catalog/model.go:37]: ok

**Comment by jianwan0214** [comment]
> Is PADDR the only system column? Others like composite primary key?
> 
> External table, may want to add columns like __mo_filename, __mo_time, etc. So this list may grow. Better keep it in one good place.

composite primary key is prefix with __mo_cpkey_ already

**Review by XuPeng-SH — APPROVED** [review]
lgtm

**Comment by fengttt** [comment]
> > Is PADDR the only system column? Others like composite primary key?
> > External table, may want to add columns like __mo_filename, __mo_time, etc. So this list may grow. Better keep it in one good place.
> 
> composite primary key is prefix with __mo_cpkey_ already

All these should be defined in one place and consistently.

**Comment by jianwan0214** [comment]
> > > Is PADDR the only system column? Others like composite primary key?
> > > External table, may want to add columns like __mo_filename, __mo_time, etc. So this list may grow. Better keep it in one good place.
> > 
> > 
> > composite primary key is prefix with __mo_cpkey_ already
> 
> All these should be defined in one place and consistently.

ok

**Review by aressu1985 — APPROVED** [review]

**Review by reusee — APPROVED** [review]

File: pkg/catalog/types.go [hunk 3]
```diff
@@ -20,7 +20,8 @@ import (
 )
 
 const (
-	Row_ID = "__rowid"
+	Row_ID           = "__mo_rowid"
+	PrefixPriColName = "__mo_cpkey_"
```
Comment (nnsgmsone) [inline pkg/catalog/types.go:24]: what is cpkey?

Comment (jianwan0214) [reply pkg/catalog/types.go:24]: this is composite primary key, I just move the position and add a prefix

**Review by nnsgmsone — APPROVED** [review]

File: pkg/sql/plan/build_show.go (hunk 2 above)
Comment (jianwan0214) [reply pkg/sql/plan/build_show.go]: solve it

**Review by fengttt — APPROVED** [review]

**Pull request merged**